conf/messages.cy

```diff
@@ -43,67 +43,69 @@ footer.links.terms_and_conditions.text=Telerau ac Amodau
+compliant.six.p.2=cysylltwch â’r Gwasanaeth Cynghori a Chymorth Cydraddoldeb
+compliant.six.p.3=http://eass-ws.custhelp.com/app/home
+compliant.six.p.4=(EASS), neu’r
+compliant.six.p.5=Equality Commission for Northern Ireland
+compliant.six.p.6=(ECNI) os ydych yn byw yng Ngogledd Iwerddon.
 
 ## section seven
-compliant.seven.heading=Contacting us by phone or getting a visit from us in person
-compliant.seven.p1=We provide a text relay service if you are deaf, hearing impaired or have a speech impediment.
-compliant.seven.p2=We can provide a British Sign Language (BSL) interpreter, or you can arrange a visit from an HMRC advisor to help you complete the service.
-compliant.seven.p3.1=Find out how to
-compliant.seven.p3.2=contact us
+compliant.seven.heading=Cysylltu â ni dros y ffôn neu gael ymweliad personol gennym
+compliant.seven.p1=Rydym yn cynnig gwasanaeth text relay os ydych yn fyddar, â nam ar eich clyw neu os oes gennych nam ar eich lleferydd.
+compliant.seven.p2=Gallwn ddarparu dehonglydd Iaith Arwyddion Prydain (BSL), neu gallwch drefnu ymweliad gan ymgynghorydd CThEM i’ch helpu i gwblhau’r gwasanaeth.
+compliant.seven.p3.1=Gwybodaeth am sut i
+compliant.seven.p3.2=gysylltu â ni
+compliant.seven.p3.3=https://www.gov.uk/cael-help-cthem-cymorth-ychwanegol
 
 ## section eight
-compliant.eight.heading=Technical information about this service’s accessibility
-compliant.eight.p1=HMRC is committed to making this service accessible, in accordance with the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.
-compliant.eight.p2.1=This service is fully compliant with the
-compliant.eight.p2.2=Web Content Accessibility Guidelines version 2.1 AA standard
+compliant.eight.heading=Gwybodaeth dechnegol am hygyrchedd y gwasanaeth hwn
+compliant.eight.p1=Mae CThEM wedi ymrwymo i wneud y gwasanaeth hwn yn hygyrch, yn unol â Rheoliadau Hygyrchedd Cyrff Sector Cyhoeddus (Gwefannau a Chymwysiadau Symudol) (Rhif 2) 2018.
+compliant.eight.p2.1=Mae’r gwasanaeth hwn yn cydymffurfio’n llawn â
+compliant.eight.p2.2=safon ‘AA’ Canllawiau Hygyrchedd Cynnwys y We, fersiwn 2.1
 
 ## section nine
-compliant.nine.heading=How we tested this service
-compliant.nine.p1=The service was launched on 20 April 2020 and checked for compliance with WCAG 2.1 AA.
-compliant.nine.p2.1=The service was built using parts that were tested by the
+compliant.nine.heading=Sut gwnaethom brofi’r gwasanaeth hwn
+compliant.nine.p1=Cafodd y gwasanaeth ei brofi ddiwethaf ar 5 Awst 2020 a gwiriwyd ei fod yn cydymffurfio â safon ‘AA’ Canllawiau Hygyrchedd Cynnwys y We, fersiwn 2.1.
+compliant.nine.p2.1=Adeiladwyd y gwasanaeth gan ddefnyddio rhannau a brofwyd gan y Ganolfan Hygyrchedd Digidol (
```

Comment:
Rendered content has a space either side of the (Accessibility Centre) brackets as below - document doesn't have spaces

> Adeiladwyd y gwasanaeth gan ddefnyddio rhannau a brofwyd gan y Ganolfan Hygyrchedd Digidol (Digital Accessibility Centre). Profwyd y gwasanaeth llawn gan CThEM, ac roedd y gwaith o brofi’r gwasanaeth yn cynnwys defnyddwyr anabl.